src/main/java/ch/naviqore/gtfs/schedule/model/Stop.java

```diff
@@ -17,6 +18,7 @@ public final class Stop implements Initializable, Location<GeoCoordinate> {
 
     private final String id;
     private final String name;
+    @Nullable private final String parentId;
```

Comment:
I would assign the stop directly instead of just the id. Otherwise we need to lookup everytime the stop when we need it. It is also more memory efficient to share one reference instead of repeating the string of the id for every child.

Reply:
Probably agree. Then I would also replace the Hashmap key from string to Stop in the service in NAV-87.